### PR TITLE
feat: PATRON2020-424 add support for hvacStatus, hvacRoom and led not…

### PR DESCRIPTION
**Pull request by kamiljalowiczor**
…ifications

### CODE REVIEW

**Comment by jwieckowski** [comment]
Is this possible to remove displaying the id in HVACStatus type notifications?

**Comment by MartaJaszewska** [comment]
> Is this possible to remove displaying the id in HVACStatus type notifications?

IMO id should be displayed, because it is probably id of the room in which the heating/cooling works.

File: frontend/src/components/Dashboard/SmartHomeMap/Map/Sensor/SensorGraphicComponent.jsx [hunk 1]
```diff
@@ -17,7 +19,10 @@ export function drawSensorGraphicComponent (sensorType, sensorData) {
     RFIDSensor: <RFIDSensorIcon />,
     smokeSensor: <SmokeSensorIcon isSmokeDetected={sensorData && sensorData.isSmokeDetected} />,
     LED_CONTROLLER: <LightSensorIcon />,
-    TEMPERATURE_SENSOR_ICON: <TemperatureSensorIcon temperature={sensorData && sensorData.value} />
+    led: <LightSensorIcon />,
+    TEMPERATURE_SENSOR_ICON: <TemperatureSensorIcon temperature={sensorData && sensorData.value} />,
+    hvacRoom: <HvacRoomIcon />,
```
Comment (MartaJaszewska) [inline frontend/src/components/Dashboard/SmartHomeMap/Map/Sensor/SensorGraphicComponent.jsx:24]: It should be HVACRoom and HVACStatus.

**Comment by kamiljalowiczor** [comment]
> > Is this possible to remove displaying the id in HVACStatus type notifications?
> 
> IMO id should be displayed, because it is probably id of the room in which the heating/cooling works.

actually since HVACStatus always has id of -1 maybe its better to just not display it at all. seems like there is only one HVACStatus anyways

File: frontend/src/components/Dashboard/SmartHomeMap/Map/Sensor/SensorGraphicComponent.jsx (hunk 1 above)
Comment (kamiljalowiczor) [reply frontend/src/components/Dashboard/SmartHomeMap/Map/Sensor/SensorGraphicComponent.jsx:24]: not really, this is exact notification type we get from embedded team

**Pull request merged**